README.md

```diff
@@ -1,40 +1,37 @@
-# Signal Android 
+# Signal Android Websockets-Only
```

Comment:
I'd like to not call this that way, because of pull https://github.com/JavaJens/TextSecure/pull/40, which I still think should be merged. Also it's written ``WebSocket`` :).

Why not use ``Signal Android with WebSocket support``?